### ISSUE TITLE
"Low FPS"

### ISSUE THREAD

**Issue by cauehsf** [LibreSprite/LibreSprite#33]
### The Issue ###
I use the Flatpak version of the [Libresprite](https://flathub.org/apps/details/com.github.libresprite.LibreSprite), and everything works fine, until i open a new Canvas, then the program starts to "run in low fps" (I don't know how to explain this.), i can brush, can draw, i can do every single thing, but it's totally "lagged"

[Example GIf](https://gph.is/g/46b1KY4)

Edit:
It stops the "low fps" when running on 100% zoom level.

### Temporary Fix  ###
100% Screen Scaling
200% UI Elements Scaling (It looks like 200% Screen Scaling, fixes the "low fps")

### LibreSprite and System version

* LibreSprite: 1.0-dev
* System: Linux Mint 19.1 (AMD) (I never tested on other Distros. Maybe i'll on Ubuntu 19.04.)

**Comment by zeta0134** [comment]
Confirming the issue and the workaroung. I didn't have this issue before I switched GPUs. I'm running Arch Linux and building LibreSprite: 1.0-dev, clean from the latest master. I pass no flags to ninja / cmake during the build, so it uses whatever the defaults are.

Prior GPU, no lag: nVidia GT 730
Current GPU, with lag: AMD Radeon VII

While mousing around, I noticed that Xorg was consuming 100% of my CPU, which suggests that the method LibreSprite is using to do screen scaling is possibly hitting a performance problem in the AMD driver. That's an educated guess though, I really don't know what's going on.
